src/uploader.js

```diff
@@ -101,7 +101,7 @@ const useWidget = (
       widget.current.onUploadComplete.remove(changeCallback)
       widget.current.onChange.remove(fileSelectedCallback)
     }
-  }, [changeCallback, fileSelectedCallback])
+  }, [changeCallback, fileSelectedCallback, uploadcare, attributes])
```

Comment:
I think enough only 'attributes' in deps, without 'uploadcare'.

Reply:
@FilippovAM Probably you're right, but a [few lines below](https://github.com/uploadcare/react-widget/blob/059c59395d2ee9f817c19b379f3f3eb6987c8b7c/src/uploader.js#L92) there are `uploadcare` and `attributes` dependencies at the widget initialization effect. I don't know why, because it seems that uploadcare object reference is always static, maybe Dima Ivakhnenko did it for a some reason. So I think it's better to have equal dependencies at those effects just to be sure that they will be executed consistently and to avoid situations when callbacks aren't added to the new widget instance.